Dockerfile.app

```diff
@@ -1,28 +1,27 @@
 ARG NODE_VERSION="8.9.2"
 ARG NPM_VERSION="5.5.0"
-FROM registry.devshift.net/fabric8-hdd/nodejs:${NODE_VERSION}_npm_${NPM_VERSION}
-LABEL maintainer="Anmol Babu <anmolbudugutta@gmail.com>"
-
-# Setup App Dir
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+ARG OS_REGISTRY="registry.devshift.net/fabric8-hdd"
```

Comment:
can we call this just REGISTRY or probably APP_REGISTRY I think this name and the one in https://github.com/fabric8-hdd/fabric8-hypothesis-core-image/pull/16/files#diff-3254677a7917c6c01f55212f86c57fbfR5 need not be the same(although they are most likely to be the same)